gui/src/shared/components/kg/node/KgNodeLink.tsx

```diff
@@ -5,13 +5,13 @@ import {kgId} from "shared/api/kgId";
 
 export const KgNodeLink: React.FunctionComponent<{
   node: {id: string; label: string | null; pos: string | null};
-  datasource?: string;
-}> = ({node, datasource}) => {
+  sources?: string[];
+}> = ({node, sources}) => {
   return (
     <Link data-cy="node-link" to={Hrefs.kg({id: kgId}).node({id: node.id})}>
       {(node.label ? node.label : node.id) +
         (node.pos ? " (" + node.pos + ")" : "") +
-        (datasource ? " - " + datasource : "")}
+        (sources && sources.length > 0 ? " - " + sources.join(",") : "")}
```

Comment:
Nit: should probably use | as the join string to be consistent with KGTK conventions.

Reply:
Changed to |